src/theme/DocPage/styles.module.css

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+:root {
```

Comment:
No edits made to this file - the stylesheets get ejected along with the corresponding component.